platform-includes/distributed-tracing/custom-instrumentation/javascript.mdx

```diff
@@ -173,6 +173,28 @@ In this example, tracing information is propagated to the project running at `ht
 
 The two services are now connected with your custom distributed tracing implementation.
 
+### Starting a New Trace
+
+In case the SDK's [default behavior](../#trace-duration) for the trace duration does not fir your use case, you can manually start a new trace that will no longer be connected to the current (distributed) trace.
```

Comment:
```suggestion
In case the SDK's [default behavior](../#trace-duration) for the trace duration does not fit your needs, you can manually start a new trace that will no longer be connected to the current (distributed) trace.
```